src/main/java/gregtech/common/metatileentities/multi/electric/generator/MetaTileEntityLargeTurbine.java

```diff
@@ -201,4 +203,9 @@ public boolean isStructureObstructed() {
     public int getTier() {
         return tier;
     }
+
+    @Override
+    public boolean canVoidRecipeOutputs() {
+        return true;
+    }
```

Comment:
You should remove the override in `LargeTurbineWorkableHandler` if you are adding this here